termenv_unix.go

```diff
@@ -57,6 +60,22 @@ func (o *Output) ColorProfile() Profile {
 	}
 
 	if strings.Contains(term, "256color") {
+		// Check for WSL
+		wsl, err := ioutil.ReadFile("/proc/sys/kernel/osrelease")
+		// Additional check for Mac OS as it doesn't have "/proc/" folder
+		if err != nil && !errors.Is(err, os.ErrNotExist) {
+			return ANSI256
+		}
+		if string(wsl) != "" {
+			// Lowercasing every content inside "/proc/sys/kernal/osrelease"
+			// because it gives "Microsoft" for WSL and "microsoft" for WSL 2
+			// so no need of checking twice
+			wslLower := strings.ToLower(string(wsl))
+			// Also check if the terminal used is Windows Terminal
+			if strings.Contains(wslLower, "Microsoft") && os.Getenv("WT_SESSION") != "" {
```

Comment:
```suggestion
			if strings.Contains(wslLower, "microsoft") && os.Getenv("WT_SESSION") != "" {
```
Otherwise it will always be false :)

Reply:
Oops :sweat_smile: